Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/IncidentProducer/IncidentProducer.cs

```diff
@@ -28,80 +17,139 @@ namespace Helios2Sentinel
 {
     public class IncidentProducer
     {
-        private static string keyVaultName = Environment.GetEnvironmentVariable("keyVaultName");
-        private static string azureWebJobsStorage = Environment.GetEnvironmentVariable("AzureWebJobsStorage");
+        private static readonly string keyVaultName = Environment.GetEnvironmentVariable("keyVaultName");
+        private static readonly string azureWebJobsStorage = Environment.GetEnvironmentVariable("AzureWebJobsStorage");
         private static readonly object queueLock = new object();
-        private static string containerName = "cohesity-extra-parameters";
+        private static readonly string containerName = "cohesity-extra-parameters";
+        private static readonly string lastRequestDetailsBlobKey = Environment.GetEnvironmentVariable("Workspace") + "\\last-request-details";
+        private static readonly string requestDetailsAlertIdKey = "alertIds";
+        private static readonly HashSet<String> alertPropertiesRequired =
```

Comment:
Where do you use alertPropertiesRequired? I think that you can remove it

Reply:
Nice catch! Added it at the place where it's required now.